unit_conversion

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+#
+#                         License
+#
+# Copyright (C) 2024  David Valin dvalin@redhat.com
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# Convert various time and memory units.
+
+usage()
+{
+	echo "usage $1 <unit>"
+	echo "Known time units"
+	echo "	hour/hr/h, minute/min/m, sec, msec, ms, usec, us, nsec, ns"
+	echo "Known memory units"
+	echo "	byte, B, Kib, KB, MiB, MB, GiB, GB"
+	exit 1
+}
+
+#
+# Coverts units to base units.
+# In case of time, that will be nanoseconds
+# In case of throughput, that will be bytes.
```

Comment:
Would it be possible to allow a second arg to act as the `to` unit?  This will make it more flexible for future usage.

Reply:
Agree with @kdvalin , especially since while nanoseconds may be fairly future-resistant bytes for throughput is going to make for some very large numbers in charts. What's good for a filesystem test may not be good for a network test.